test/mason/publish-tests/publishGitCheck.chpl

```diff
@@ -0,0 +1,21 @@
+use FileSystem;
```

Comment:
You could confirm it by testing these in a clean environment, but I'm pretty sure we'll need all the standard mason `EXECENV`, `SKIPIF` files here. (recall we symlink to reduce code redundancy)